subprojects/docs/src/main/resources/header.html

```diff
@@ -113,6 +113,7 @@ <h3 id="using-gradle-builds">Running Gradle Builds</h3>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Using Build Scans</a></li>
             <li><a href="/userguide/build_cache.html">Enabling and Configuring the Build Cache</a></li>
             <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
+            <li><a href="/userguide/directory_layout.html">Directory Layout</a></li>
```

Comment:
This deserves a more prominent place IMO. Definitely above "Executing Multi-Project Builds" and maybe should be part of "Getting Started". Do you agree @pledbrook?

Reply:
I think this section of the nav is already overcrowded, and as I mention above, I don't see the value of having this so prominent. Also, I don't think "Directory Layout" is a sufficient chapter title as it could easily be interpreted — and more likely to be interpreted in my view — as about a _build_'s or _project_'s directory layout.

As I mention on [issue #262](https://github.com/gradle/dotorg-docs/issues/262), I think details of the internals of Gradle, such as the wrapper, daemon, and the working directories it uses should be aggregated into an appendix, or perhaps their own reference section.

Reply:
@pledbrook Do you have a proposal for a better chapter title?

Reply:
No really good ones. "Gradle's hidden directories/files"? "Gradle's directories and files"? "Gradle's caches and other working files"? "The directories Gradle uses for internal purposes"?!

I don't know if there is a good, single word that describes what these directories and files are. I'm sure there must be a technical term for such files that's well known, but nothing's popping into my head.